app/(blobs)/(setup)/_components/SignUpForm.tsx

```diff
@@ -18,6 +19,7 @@ export const SignUpForm = () => {
 
   const onSubmit = async (data: unknown) => {
     await signup(data);
+    redirect('/setup?step=2');
```

Comment:
Using redirect here throws a runtime error. Next [docs](https://nextjs.org/docs/app/building-your-application/routing/redirecting#userouter-hook) recommend using `useRouter` and `router.push()` in client components. Any reason not to here?

Reply:
potential approach with `useRouter` instead here [9cca019](https://github.com/complexdatacollective/Fresco/pull/162/commits/9cca019e3b866a31ee57b830f2abdd5e67bef01e)